generate_packages.py

```diff
@@ -206,7 +211,7 @@ def main():
         metas[pkg.name] = meta
 
         # Aliases can be linked too
-        for alias in raw_aliases:
+        for alias in sorted(list(raw_aliases)):
```

Comment:
Just a comment, but I think this is not needed:
```suggestion
        for alias in sorted(raw_aliases):
```